patches/chromium/ssl_security_state_tab_helper.patch

```diff
@@ -39,7 +44,19 @@ index 887cdd11fbbd9df44edef1548a89891c5cf23cee..12da72449622289ec467d284f616bbf9
  #include "chrome/browser/safe_browsing/chrome_password_protection_service.h"
  #endif
  
-@@ -66,8 +70,9 @@ void RecordSecurityLevel(
+@@ -70,6 +74,7 @@ void RecordSecurityLevel(
+   }
+ }
+ 
++#if 0
```

Comment:
I don't see a reason why the warning of TLS deprecation utility should be excluded. The components are all in place and the only additional source needed is `tls_deprecation_config.h`

Reply:
The `IsLegacyTLS` function uses `g_browser_process` which we'd previously excluded, so I took it out. Without `IsLegacyTLS`, `SSLProtocolVersionFromString` is otherwise unused.

Reply:
We mock the `g_browser_process` here https://github.com/electron/electron/blob/master/shell/browser/browser_process_impl.cc#L33 , so its harmless to include it. The place where we excluded was using a preference we don't set, hence it was excluded.

Here `prefs::kSSLVersionMin` is a preference that is loaded from command line https://cs.chromium.org/chromium/src/chrome/browser/prefs/chrome_command_line_pref_store.cc?l=123, we support this particular command line switch, hence I would suggest to parse the command line directly in `IsLegacyTLS` instead of reading pref value to retain the functionality.